src/components/pullRequests/pullRequests.module.scss

```diff
@@ -0,0 +1,57 @@
+.pullRequest {
+    border: 0.2em solid black;
+    border-radius: 0.5em;
+    padding: 1em;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin: 2em;
```

Comment:
Please use `px` as a unit for border, margin, padding, these metrics are not dependent on the parents/roots font-size.
https://css-tricks.com/rems-ems/